Documentation/ApiOverview/TypoScriptSyntax/Syntax/Conditions.rst

```diff
@@ -274,12 +275,123 @@ from the top level. This is normally done when TypoScript code from
 various records is combined.
 
 
+.. index:: TypoScript; Symfony expression language
 .. _typoscript-syntax-conditions-expression-language:
 
-Custom Conditions With Symfony Expression Language
-==================================================
+Custom conditions with the `Symfony`:pn: expression language
+============================================================
+
+Further information about how to extend TypoScript with your own custom
+conditions can be found within :ref:`sel-within-typoscript-conditions`.
+
+
+.. _typoscript-syntax-conditions-migration:
+
+Migration from the pre-`Symfony`:pn: condition syntax
```

Comment:
Additionally I would add information about when you must migrate (which TYPO3 version), e.g. by mentioning when this was deprecated, possibly with a link to the changelog.